src/jiraCommon.gs

```diff
@@ -577,41 +546,11 @@ function unifyIssueAttrib(attrib, data) {
       resp.value = data[attrib] || data.fields[attrib];
       break;
   }
-  } catch (e) {}
-  
-  return resp;
-}
-
-/**
- * @desc Return table header title for issue property
- * @param header {string}  Property key name to get header title for
- * @return {string}
- */
-function headerNames(header) {
-  var label, labels = ISSUE_COLUMNS;
-  extend(labels, {
-    key: 'Key',
-    issuetype: 'Type',
-    duedate: 'Due',
-    priority: 'P',
-  });
-
-  // append favorite custom fields
-  extend(labels, getCustomFields(CUSTOMFIELD_FORMAT_SEARCH));
-
-  // custom epic
-  var epicField = UserStorage.getValue('jst_epic');
-  if (epicField.usable === true) {
-    labels[epicField.link_key] = 'Epic';
+  } catch (e) {
+    debug.log("Error in unifyIssueAttrib "+e);
```

Comment:
This try catch made it difficult to unit test so I thought it useful both for the testing and the production site to log errors